qlog_test.go

```diff
@@ -16,18 +18,14 @@ type nopCloser struct{}
 func (nopCloser) Close() error { return nil }
 
 var _ = Describe("qlogger", func() {
-	var origQlogDir string
+	var qlogDir string
 
 	BeforeEach(func() {
-		origQlogDir = qlogDir
-		d, err := ioutil.TempDir("", "libp2p-quic-transport-test")
+		var err error
+		qlogDir, err = ioutil.TempDir("", "libp2p-quic-transport-test")
```

Comment:
We should probably do the equivalent of `t.Cleanup()` to delete the temp dir after the test is done.

Reply:
Done.